benchmarks/utils/update_artifacts.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import os
+import shutil
+
+BENCHMARK_REPORT_PATH = "/tmp/ts_benchmark"
+BENCHMARK_ARTIFACTS_PATH = "/tmp/ts_artifacts"
+BENCHMARK_REPORT_FILE = "ab_report.csv"
+WINDOW_LEN = 8
+WINDOW_START = 0
+
+################################################################
+# This is an example directory structure for the artifacts.
+# Here, report_id 1 is missing, new report would be added under 1
+# and we would remove report_id 2.
+# .
+# └── tmp/
+#    └── ts_artifacts/
+#        ├── 0/
+#        │   ├── eager_mode_mnist_w4_b1/
+#        │   │   └── ab_report.csv
+#        │   ├── eager_mode_mnist_w4_b2/
+#        │   │   └── ab_report.csv
+#        │   └── ...
+#        ├── 2/
+#        │   ├── eager_mode_mnist_w4_b1/
+#        │   │   └── ab_report.csv
+#        │   ├── eager_mode_mnist_w4_b2/
+#        │   │   └── ab_report.csv
+#        │   └── ...
+#        ├── 3/
+#        │   ├── eager_mode_mnist_w4_b1/
+#        │   │   └── ab_report.csv
+#        │   ├── eager_mode_mnist_w4_b2/
+#        │   │   └── ab_report.csv
+#        │   └── ...
+#        ├── ...
+#        └── 6/
+#            ├── eager_mode_mnist_w4_b1/
+#            │   └── ab_report.csv
+#            ├── eager_mode_mnist_w4_b2/
+#            │   └── ab_report.csv
+#            └── ...
+################################################################
+
+
+# Copy BENCHMARK_REPORT_FILE to artifacts
+def copy_benchmark_reports(input, output):
+
+    for dir in os.listdir(input):
+        if os.path.isdir(os.path.join(input, dir)):
+            new_dir = os.path.join(output, dir)
+            os.makedirs(new_dir, exist_ok=True)
+            shutil.copy(os.path.join(input, dir, BENCHMARK_REPORT_FILE), new_dir)
+
+
+# Save new report and delete the oldest report
+def update_new_report(input_dir, output_dir, add_report_id, del_report_id):
+
+    # Add new report
+    new_dir = os.path.join(output_dir, str(add_report_id))
```

Comment:
Nit: Would it make sense to sanity check if `add_report_id` is an `int` and use `add_report_id % WINDOW_LEN`?